ibis/sql/alchemy.py

```diff
@@ -509,6 +509,120 @@ def build_ast(expr, context=None, dialect=None):
     return builder.get_result()
 
 
+class AlchemyDatabase(Database):
+    """
+    
+    Attributes
+    ----------
+    client : AlchemyClient
+    
+    """
+
+    def __init__(self, name, client):
+        super().__init__(name, client)
+
+    def table(self, name, schema=None):
+        self.client.table(name, schema=schema)
+
+    def list_tables(self, like=None, schema=None):
+        return self.client.list_tables(
+            schema=schema,
+            like=self._qualify_like(like),
+            database=self.name)
+
+class AlchemyDatabaseSchema(object):
```

Comment:
I can probably just derive from Database for this.  Not sure if we want to make some kind of base class for Database/Schema that is most of what Database is now?